model_templates/training/python3_sklearn_transform/README.md

```diff
@@ -8,8 +8,7 @@ This transformer is intended to work with the [Python 3 Scikit-Learn Drop-In Env
 
 Note that transformers differ from custom models in that the end product is not a trained model, but a transformation of the dataset.
 As such, the fit method will train only a preprocessing pipeline on your data. The transform hook must be implemented by the user for now.
-You will also have to pass target info, or the --unsupervised flag if there is no target in the downstream modeling task that will be used by
-this transformer.
+You will also have to pass target info according to the downstream modeling task that will be used by this transformer.
```

Comment:
we should probably make sure you can run a transformer with or without a target specified

Reply:
updated condition in `verify_options`